src/osd/OSD.h

```diff
@@ -581,6 +581,7 @@ class OSDService : public Scrub::ScrubSchedListener {
 
   void queue_for_pg_delete(spg_t pgid, epoch_t e);
   bool try_finish_pg_delete(PG *pg, unsigned old_pg_num);
+  int64_t get_scrub_cost();
```

Comment:
Is there a reason to make this public?

Reply:
Nope, it doesn't have to be. I'll make the change